shared/editor/embeds/Diagrams.tsx

```diff
@@ -10,6 +11,16 @@ function Diagrams(props: Props) {
     ? `Diagrams.net (${params.get("title")})`
     : "Diagrams.net";
 
+  const integration = find(
+    props.integrations.orderedData,
```

Comment:
Not sure why `.orderedData` is needed here. Already passing normalized `integrations` [here](https://github.com/outline/outline/pull/3980/commits/c5a5c14089e476c661d454fa700e7507e6f46f7e#diff-18ef7e911d2802727b4354c37055ae38684c757f7de708d9010180f2ea98d248R179)

Reply:
Hmm, yea that doesn't seem to make any sense to me either

Reply:
Just realized that this is because of the name collision with `RootStore.integrations` [here](https://github.com/outline/outline/pull/3980/commits/c5a5c14089e476c661d454fa700e7507e6f46f7e#diff-4e4902c1514c28c9a67dea9579999973b380d7591a46c22f1574d7ebdde31389R78) :laughing: